com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/lightlistbuild-clustered.compute

```diff
@@ -12,6 +12,11 @@
 
 // #pragma enable_d3d11_debug_symbols
 
+#ifdef SHADER_API_XBOXONE
```

Comment:
don't use #ifdef SHADER_API_XBOXONE  :)   that's all the point of what we discuss